greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md

```diff
@@ -9,4 +9,6 @@
 
 [Ανάβεις τον αναπτήρα](set_it_on_fire/set_it_on_fire.md)
 
-[Ανάβεις ακόμη ένα τσιγάρο](intimacy/intimacy.md)
\ No newline at end of file
+[Ανάβεις ακόμη ένα τσιγάρο](intimacy/intimacy.md)
+
+[Μπορείς ακόμα να βγάλεις την κάλτσα σου] (put_socks_off/put-socks-off.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.